Packages/com.chisel.editor/Chisel/Editor/Components/Generators/CSGCapsule/Editor/CSGCapsuleEditor.cs

```diff
@@ -148,12 +148,14 @@ protected override void OnScene(CSGCapsule generator)
 			UnityEditor.Handles.color = CSGCylinderEditor.GetColorForState(baseColor, false, true, isDisabled);
 			DrawOutline(generator.definition, vertices, lineMode: LineMode.NoZTest);
 			
-
 			var topPoint	= normal * generator.Height;
 			var bottomPoint = normal * 0;
 			var middlePoint	= normal * (generator.Height * 0.5f);
 			var radius2D	= new Vector2(generator.definition.diameterX, generator.definition.diameterZ) * 0.5f;
 
+            if (generator.Height < 0)
+                normal = -normal;
```

Comment:
prevents handle arrows pointing in the wrong direction when the height is inverted